src/site/resources/styleguides/google-java-style-20220203/javaguide.html

```diff
@@ -380,6 +402,8 @@ <h3 id="s4.4-column-limit">4.4 Column limit: 100</h3>
   3.2 <a href="#s3.2-package-statement">Package statement</a> and
   3.3 <a href="#s3.3-import-statements">Import statements</a>).</li>
 
+  <li>Contents of <a href="s4.8.9-text-blocks">text blocks</a>.</li>
+
```

Comment:
text blocks are ok to skip linelength, I remember we did some work on it, please share a link how we covered this diff